app/views/cloud_volume/_common_new_edit.html.haml

```diff
@@ -49,6 +49,20 @@
     %span.help-block{"ng-show" => "angularForm.aws_availability_zone_id.$error.required"}
       = _("Required")
 
+.form-group{"ng-if"    => "vm.cloudVolumeModel.emstype == 'ManageIQ::Providers::Amazon::StorageManager::Ebs'"}
+  %label.col-md-2.control-label
+    = _('Cloud Snapshot')
+  .col-md-8
+    %select{"name"        => "aws_cloud_volume_snapshot_id",
+            "ng-model"    => "vm.cloudVolumeModel.aws_cloud_volume_snapshot_id",
+            "ng-options"  => "vs.ems_ref as vs.name for vs in vm.cloudVolumeSnapshotChoices",
+            "ng-disabled" => "!vm.newRecord",
+            :checkchange  => true,
+            "data-live-search" => "true",
+            "pf-select"   => true}
+      %option{"value" => "", "enabled" => ""}
+        = "#{_('No snapshot')}"
```

Comment:
You can change this to `= _('No snapshot')` since you are not rendering any other chars, like `<` and `>` in other cases.